Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/BXLInvocationAnalyzer.cs

```diff
@@ -78,10 +85,16 @@ internal sealed class BXLInvocationAnalyzer : Analyzer
         public override int Analyze()
         {
             var dataStore = new XldbDataStore(storeDirectory: InputDirPath);
-
-            File.WriteAllLines(OutputFilePath, dataStore.GetEventsByType(Xldb.ExecutionEventId.DominoInvocation));
+            File.AppendAllLines(OutputFilePath, dataStore.GetBXLInvocationEvents());

```

Comment:
Just checking if this is intentional , or just left from debugging.... This will be an ever growing file over time not?

Reply:
This is intentional, and the file should not be growing over time as you should only be calling this "analyzer" once.